DiceKeys/Core/DiceKey/Derivables.swift

```diff
@@ -49,10 +49,19 @@ func addLengthInCharsToRecipeJson(_ recipeWithoutLengthInChars: String, lengthIn
 }
 
 func addSequenceNumberToRecipeJson(_ recipeWithoutSequenceNumber: String, sequenceNumber: Int) -> String {
-    guard sequenceNumber != 1 else { return recipeWithoutSequenceNumber }
+    guard sequenceNumber > 1 else { return recipeWithoutSequenceNumber }
     return addFieldToEndOfJsonObjectString(recipeWithoutSequenceNumber, fieldName: "#", fieldValue: String(describing: sequenceNumber))
 }
 
+// This is a temporary solution. Refactor is needed as per Android implementation
```

Comment:
Please create an issue to ensure this gets fixed and refer to it here.  Then merge